src/clients/algosigner.ts

```diff
@@ -92,18 +92,17 @@ class AlgoSignerClient extends BaseWallet {
 
     let ledger: SupportedLedgers = "MainNet";
 
-    /** @todo remove env vars */
-    // if (process.env.NEXT_PUBLIC_VERCEL_ENV === "betanet") {
-    //   ledger = "BetaNet";
-    // }
+    if (process.env.NEXT_PUBLIC_VERCEL_ENV === "betanet") {
+      ledger = "BetaNet";
+    }
 
-    // if (process.env.NEXT_PUBLIC_VERCEL_ENV === "testnet") {
-    //   ledger = "TestNet";
-    // }
+    if (process.env.NEXT_PUBLIC_VERCEL_ENV === "testnet") {
+      ledger = "TestNet";
+    }
 
-    // if (!process.env.NEXT_PUBLIC_VERCEL_ENV) {
-    //   ledger = "devmodenet";
-    // }
+    if (!process.env.NEXT_PUBLIC_VERCEL_ENV) {
+      ledger = "SandNet";
```

Comment:
Made this SandNet since its what I call it locally and jives with the naming scheme of others, open to changing it tho